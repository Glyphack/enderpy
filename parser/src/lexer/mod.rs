mod diagnostics;
pub mod fstring;
pub mod lexer;
