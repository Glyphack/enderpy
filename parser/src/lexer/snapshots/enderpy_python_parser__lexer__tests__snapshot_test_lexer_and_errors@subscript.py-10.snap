---
source: parser/src/lexer/mod.rs
description: "a[::d,]\n"
input_file: parser/test_data/inputs/one_liners/subscript.py
---
[
    Token {
        kind: Identifier,
        value: Str(
            "a",
        ),
        start: 0,
        end: 1,
    },
    Token {
        kind: LeftBrace,
        value: None,
        start: 1,
        end: 2,
    },
    Token {
        kind: Colon,
        value: None,
        start: 2,
        end: 3,
    },
    Token {
        kind: Colon,
        value: None,
        start: 3,
        end: 4,
    },
    Token {
        kind: Identifier,
        value: Str(
            "d",
        ),
        start: 4,
        end: 5,
    },
    Token {
        kind: Comma,
        value: None,
        start: 5,
        end: 6,
    },
    Token {
        kind: RightBrace,
        value: None,
        start: 6,
        end: 7,
    },
    Token {
        kind: NewLine,
        value: None,
        start: 7,
        end: 8,
    },
]
